src/middlewared/middlewared/plugins/catalogs_linux/items.py

```diff
@@ -70,6 +70,8 @@ def get_trains(self, location, options=None):
             category_path = os.path.join(location, train)
             for item in filter(lambda p: os.path.isdir(os.path.join(category_path, p)), os.listdir(category_path)):
                 item_location = os.path.join(category_path, item)
+                if (not os.path.isdir(item_location) or train.startswith('.')):
```

Comment:
This is not required as the `filter` above at L71 already ensures that at this stage `item_location` is a directory. Please let me know otherwise.

Reply:
ping @Ornias1993